lib/actions-oclif/device/index.ts

```diff
@@ -119,6 +172,17 @@ export default class DeviceCmd extends Command {
 				'note',
 				'os_version',
 				'dashboard_url',
+				'cpu_usage_percent',
+				'cpu_temp_c',
+				'cpu_id',
+				'memory_usage_mb', // Use MiB, when we support lowercase in headers
```

Comment:
I see. But having `MEMORY USAGE MIB` is probably better than `MEMORY USAGE MB`, no? I mean, at least users don't get confused with 1,000 vs 1,024...

Reply:
The YouTube poll has settled it as MB.